libqtile/widget/base.py

```diff
@@ -29,30 +29,38 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from .. import command, bar, configurable, drawer
+from .. import command, bar, configurable, drawer, confreader
 import six
 import subprocess
 import logging
 import threading
 import warnings
 
+ORIENTATION_NONE = 0
+ORIENTATION_HORIZONTAL = 1
+ORIENTATION_VERTICAL = 2
+ORIENTATION_BOTH = 3
```

Comment:
Can we export these in libqtile.config, so that that's still the module everyone looks in to get configuration things? (We can either move them there or just re-export them, either way is fine with me.)